content/archive/internal/yiugn79-tatcontrosheet.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "TAT 관리"
+author: "yiugn79"
+date: "2021-03-10T08:49:40Z"
+link: "https://bookdown.org/yiugn79/tatcontrosheet/"
+length_weight: "0%"
+pinned: false
+---
+
+NA ...
```

Comment:
What's going on with this one?

Reply:
It is not book - just a HTML file with no text content. 

I'll see why it does not get filtered out